### DISCUSSION TITLE
Welcome to Apex-NoRecoil-2021 Discussions!

### DISCUSSION THREAD

**Discussion by mgsweet** [mgsweet/Apex-NoRecoil-2021#166] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zhangjia701** [comment]
I think we can add a resolution of 2304 * 1440. This is a 2K 16:10. The size of the characters in the game will expand

**Comment by BamaBinBombin** [comment]
Idea for discussions subgroup:  post modified(improved) recoil pattern .txt files...  Anyone care to do this?
I've improved on a few doing hours worth of recordings and viewing in slow-motion with doing changes and reviews, improved a few a lot, also made some crazy modifications that actually worked, but anyone else doing this too and want to share?

**Reply by NicktheQuick33** [reply]
Would be great to get these dude!
I've noticed 1.3.10 Alternator pattern drags down a lot. 1.3.6 seems to be the best.

**Reply by NicktheQuick33** [reply]
I've also managed to change the code to activate with my mouse scroll wheel which is ace.

**Comment by Senixisded** [comment]
Hello! I've been using the script for a while and wanna say its great! but while looking at the script's code there was a voice you could enable and I was wondering if anyone can help figure out how to make it work?

**Reply by mgsweet** [reply]
Try the script with debug mode.

**Comment by randykaskuser** [comment]
Do we have a list of weapons that can use the gold optic?
Have tried to pres 0 on alternator, C.A.R, and RE but didn't work

**Reply by BamaBinBombin** [reply]
Sorry, I have not put much time into getting the gold optics to work, but it is certainly not very clear on instructions.

**Reply by mgsweet** [reply]
I don't suggest people with no knowledge about the script's code turn on that feature. Using this feature does not result in much improvement. So if u don't know what that is, just turn it off. Not that important but more risk.

**Comment by xiaoyuyu828** [comment]
现在用这个会被ea封禁吗？求解答

**Reply by NicktheQuick33** [reply]
不，他们无法检测到此脚本

**Reply by KlarerWind** [reply]
现在会了

**Comment by yuhangwang2** [comment]
最新1.38，假如我有两把枪，第一把枪打完30发子弹（但是换子弹效果还在），换成第2把枪软件效果会消失

**Reply by yuhangwang2** [reply]
1.3.10还是会

**Reply by KlarerWind** [reply]
没有马上识别到.识别速度还是有点慢

**Comment by wurenand** [comment]
My script is not recognized and does not work

**Reply by wurenand** [reply]
我的脚本识别不了，完全无效

**Reply by GGNyny** [reply]
anyone help to fix it ?

**Comment by NicktheQuick33** [comment]
Hi Mgsweet
I'd like to have a go at fine tuning the recoil patterns.
Can you give me a basic syntax explination of the pattern scripts? What the digits mean, etc?
Thank you!!

**Reply by mgsweet** [reply]
Take a look at this repo: https://github.com/vengefulcrop/AE-Recoil-Pattern-Generation/

**Comment by BamaBinBombin** [comment]
x, y, delay between movements

Testing tip:  while reviewing in slow motion, set a large x or y value to
calculate timing
example:  instead of say 4,1,25 which would be a 4 pixels to right, then 1
pixel down with a 25ms delay, you can change that 4 to say a 400 to see
when the mouse will hit that specific line due to the large and very
obvious change to the mouse adjustments --  good to check in middle of
script or a few lines down to confirm each shot is timed correctly to the
specific line.
TIP2:  VLC slow motion makes replays blurry and hard to see clear but
replaying same clip over will clear up blurry clips by 'magic'

**Comment by InsaneUnknown** [comment]
someone can help me with creating new recoil patterns ? i dont mind doing it but im stuck after getting the coordenates on after efects. when i paste the coordnates and go training mode doesnt control the recoil , something doing wrong , ty i would like to contribute InsaneUnknown#4512  (on discord)

**Reply by NicktheQuick33** [reply]
You need to save the .txt file and completely reload the script.

**Comment by Averjax** [comment]


**Reply by NicktheQuick33** [reply]
Na, just alt-tab out of your game, right-click on the green logo in the task bar and pause the script.

**Reply by InsaneUnknown** [reply]
@mgsweet  i pay u to teach me in 30 mins . via discord . i can improve or try for all weapons if u can help me doig it . ty mate

**Reply by mgsweet** [reply]
Check this update: https://github.com/mgsweet/Apex-NoRecoil-2021/pull/227

**Comment by yuhangwang2** [comment]
此软件已被检测，一个账号封号，第2个秒封，问了4个人全部封3号

**Comment by NicktheQuick33** [comment]
Hey everyone! Looks like we have a new energy weapon coming Season 16!
Is anyone setup with slow-motion screen capture software to write the Pattern script?
@mgsweet Will you be having a go at this? Also, how do you work out the m/s delay for the pattern?

嘿大家。 看来第 16 季我们会有一种新的能量武器！
是否有人设置了慢动作屏幕捕获软件来编写模式脚本？
@mgsweet 你会尝试一下吗？ 另外，您如何计算出模式的 m/s 延迟？

**Comment by GGNyny** [comment]
Hello .. i already install the hot key and follow the guideline, cant work on my apex legend, anyone know how to fix it ?

**Comment by huazz5** [comment]
I have an error running uuid_generator.py with AHK
Regarding import uuid, why am I prompted that I did not introduce the class, and how can I resolve the problem

**Reply by huazz5** [reply]
Or should I run a.py file in python 3? I'll try it when I get home from work

**Comment by NicktheQuick33** [comment]
Which mouse sensitivity setting should we be using? Free look or ADS?

**Comment by FlashX0** [comment]
i wonder when u will support the 1366×768

**Comment by NicktheQuick33** [comment]
Hi @mgsweet 
Is there going to be an update to include the Nemesis?
Also, the script doesn't work when I run Nvidia In-Game Overlay. Any ideas why?
Thank you!

**Comment by collinsgithub601** [comment]
I got a error that said "LoadLibrary failed!
Script will now terminate"

Any advice on how to fix that?

**Comment by vodolazina** [comment]
Can you please add 3440x1440 resolution UwU?

**Reply by AlberGerke** [reply]
please add this resolution 3440*1440 21'9 UWHD

**Comment by 95276643** [comment]
1.3.11复仇女神连点的时候有bug，现在ea会不会检测到这个脚本？

**Comment by FlashX0** [comment]
Hello
can you please support in next update resolution 1366×768 and 1024×768 because i play on stretch please i know it a lot of work but i cant code and i dont understand in coding 
Thanks

**Comment by NicktheQuick33** [comment]
Nemesis intermittent firing on v1.3.11. Fine on 1.3.10. (with no recoil control, obvs).
Oh, and keep us updated on a recoil pattern for the Charge Rifle!

**Comment by Blokk65** [comment]
hey i love to play with 1440x1080  its 4:3 the same like i have at Cs GO .. i cant make yo Script propper working with 1440x1080... can u pls pls pls pleeeeeease add 1440x1080 😪

**Comment by AlberGerke** [comment]
please add this resolution 3440*1440 21'9 UWHD

**Comment by PanachaiThongpaiwan** [comment]
Hi please do the ss18 recoil gun pls

**Reply by Blokk65** [reply]
there is an update bro. just replace ne pattern folder with the updated one.

**Reply by Blokk65** [reply]
but dont use the updated ahkmaster from externalwallnut it got a few wrong written lines in it so it will fail when u run it. Stick to the old original one from mgsweet just replace the pattern folder and use the new ones

**Reply by NicktheQuick33** [reply]
Where?

**Reply by Blokk65** [reply]
Look it up its pinned on mgsweet's apex norecoil thread from eternalwallnut

**Reply by NicktheQuick33** [reply]
I can't download the entire folder, just the individual txt files. Also, Prowler etc won't work without the sub code in the main file.

**Comment by SageKatalyst** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

How can i know if it is working. I found my enemies aren't highlighted and the mouse wasn't moved directly to the enemies.

**Comment by JJturkey** [comment]
Is the author still updated?

**Comment by SageKatalyst** [comment]
i don't know






                       
原始邮件
                       
                     

发件人："huoji"< ***@***.*** &gt;;

发件时间：2023/8/28 23:04

收件人："mgsweet/Apex-NoRecoil-2021"< ***@***.*** &gt;;

抄送人："SageKatalyst"< ***@***.*** &gt;;"Comment"< ***@***.*** &gt;;

主题：Re: [mgsweet/Apex-NoRecoil-2021] Welcome to Apex-NoRecoil-2021Discussions! (Discussion #166)




 
Is the author still updated?

**Comment by POKONII** [comment]
我想知道apexmaster中的十六进制颜色代码是在哪里获取的，我无法在游戏内任何地方找到那种颜色

**Comment by n-ananananana** [comment]
Could you change it so that it also works with the L trigger and R trigger of the game controller?

**Comment by graysonman** [comment]
Does anyone have the updates r99 pattern?

**Comment by NicktheQuick33** [comment]
Any S19 updates?

**Comment by NicktheQuick33** [comment]
Alternator not working. Checked the code but nothing obvious.

**Reply by hitch3ns** [reply]
in 1080 is working , if you using diff res might be detection issue

**Reply by NicktheQuick33** [reply]
Yeah, 1440.

**Comment by imarapex** [comment]
hey can any one tell me wehere to change the image for recognition because the altinetor and the hemlock do not work after respawn added hohup on them i just cant find wehre the pixel recognation was made for ahk
